.Rbuildignore

```diff
@@ -11,3 +11,4 @@
 ^.*CITATION.cff$
 ^codecov\.yml$
 ^inst/input$
+^R/plotSankey.R$
```

Comment:
I'm guessing this is a file in your local environment that should not be included in the package? I do not know what the convention for .Rbuildignore is - at first glance it seems a bit weird to me to have a file mentioned here that is not available via the package (but no hard feelings about this on my side).

Reply:
You're right. That was a mistake. I removed it.